frontend/tailwind.config.js

```diff
@@ -1,14 +1,22 @@
 /** @type {import('tailwindcss').Config} */
+const colors = require('tailwindcss/colors');
 export default {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
   darkMode: 'media',
   theme: {
     extend: {
       colors: {
         dark: {
-          DEFAULT: 'var(--color-dark-background)',
-          textfield: 'var(--color-dark-textfield-card)',
+          DEFAULT: colors.slate[900],
+          primary: colors.slate[50],
+          secondary: colors.slate[300],
+          tertiary: colors.slate[400],
+          card: colors.slate[800],
+          title: colors.slate[50],
+          description: colors.slate[300],
+          info: colors.slate[400],
```

Comment:
@SankalpHaritash21 

The reason why I am using this syntax is because I want to access the official Tailwind colors. The Tailwind library offers convenient JavaScript color helper utilities through `tailwindcss/colors`. This utility provides an object with hex codes for all official Tailwind colors.

<details>
<summary>If you put a <code>console.log(colors)</code> somewhere, the output will resemble this: <i>(expand this block)</i></summary>
<br />

```js
// Output of console.log(colors)
{
  inherit: 'inherit',
  current: 'currentColor',
  transparent: 'transparent',
  black: '#000',
  white: '#fff',
  slate: {
    '50': '#f8fafc',
    '100': '#f1f5f9',
    '200': '#e2e8f0',
    '300': '#cbd5e1',
    '400': '#94a3b8',
    '500': '#64748b',
    '600': '#475569',
    '700': '#334155',
    '800': '#1e293b',
    '900': '#0f172a',
    '950': '#020617'
  },
  gray: {
    '50': '#f9fafb',
    '100': '#f3f4f6',
    ...
  },
  stone: {
    ...
  },
  red: {
    ...
  },
 ...
}
```

</details>

This structure is essentially an object of objects, where colors (`slate`, `red`, `cyan`, etc.) act as child keys, and their corresponding values are objects with keys represented as strings (`'50'`, `'100'`, `'200'`,...). Each of these keys have their values as different shades of the colors (`'#f8fafc'`, `'#f1f5f9'`, `'#e2e8f0'`,...).

To get a specific shade, we use the standard JavaScript object property access syntax: `obj1[childKey][key]` or **`obj1.childKey[key]`** or `obj1.childKey.key`.

For example, I'm using `colors.slate[500]` to fetch the `500` shade of the `slate` color.

Reply:
@chinmaykunkikar  have u installed tw/colors or it is inside by default

Reply:
@chinmaykunkikar now I find where I failed I tryed to use
 DEFAULT: "colors.slate" it as a string

Reply:
Thanku for this

Reply:
@SankalpHaritash21 it comes installed by default if you're using tailwind in your project.
And, IIRC you used `colors.slate` without specifying a shade (500, 300, etc).